### PR TITLE
Topic1 ex 1 2

### PR DESCRIPTION
**Pull request by julianpoma**

### CODE REVIEW

File: 01-JS-DOM-APIs/1-2/js/ex1.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function fade () {
+    setTimeout(function () {
+        document.getElementById("fade-effect").style.display = "inline"
```
Comment (maximilianotulian) [inline 01-JS-DOM-APIs/1-2/js/ex1.js]: It's fine, let me know if you want me to tell you about the standard I'm currently using.

File: 01-JS-DOM-APIs/1-2/index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <link rel="stylesheet" href="css/styles.css">
+        <script type="text/javascript" src="js/ex1.js"> </script>
+        <title></title>
+    </head>
+    <body onload="fade()">
+        <header>
+
```
Comment (maximilianotulian) [inline 01-JS-DOM-APIs/1-2/index.html]: Let's remove the tags that has not content related

File: 01-JS-DOM-APIs/1-2/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
+}.text-center {
```
Comment (maximilianotulian) [inline 01-JS-DOM-APIs/1-2/css/styles.css]: Remove the additional bracket in the begining

File: 01-JS-DOM-APIs/1-2/js/ex1.js (hunk 1 above)
Comment (julianpoma) [inline 01-JS-DOM-APIs/1-2/js/ex1.js]: Sure! Please tell me more

File: 01-JS-DOM-APIs/1-2/index.html (hunk 2 above)
Comment (julianpoma) [inline 01-JS-DOM-APIs/1-2/index.html]: Done!

File: 01-JS-DOM-APIs/1-2/css/styles.css (hunk 3 above)
Comment (julianpoma) [inline 01-JS-DOM-APIs/1-2/css/styles.css]: Done!

File: 01-JS-DOM-APIs/1-2/index.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <link rel="stylesheet" href="css/styles.css">
+        <script type="text/javascript" src="js/ex1.js"> </script>
+        <title></title>
+    </head>
+    <body onload="fade()">
+        <section class="hidden" id="fade-effect">
+            <h1>Hello World!</h1>
+        </section>
+
```
Comment (maximilianotulian) [inline 01-JS-DOM-APIs/1-2/index.html]: Just a minor thing, the only blank line we've is at line 14, should we keep it or remove it?

File: 01-JS-DOM-APIs/1-2/js/ex1.js (hunk 1 above)
Comment (maximilianotulian) [inline 01-JS-DOM-APIs/1-2/js/ex1.js]: Fine! Let's try to not use global definitions. Create a variable instead. In this particular exercise won't break anything. If the environment we're working is bigger a global definition could have side effects. Don't need to apply it here

File: 01-JS-DOM-APIs/1-2/index.html (hunk 4 above)
Comment (julianpoma) [inline 01-JS-DOM-APIs/1-2/index.html]: I 'd keep it for better visibility of the code. It doesnt hurt

**Pull request merged**